radar/infrastructure_config/teamcity.yml

```diff
@@ -0,0 +1,16 @@
+version: 1
+name: TeamCity
+blip:
+  - version: "2.6"
+    ring: ADOPT
```

Comment:
Not crazy about `ADOPT` for in-house build servers.

Reply:
I think in current state this is the only way to build new RS services so it needs to be adopt until Productivity team have come up with a new solution

Reply:
We just analyzed Jenkins and TeamCity, and we discovered that TeamCity was the best fit as the platform look today. We are going to continue to use TeamCity for the time being.

Reply:
Let's keep this as-is in _this_ pull request and open a separate discussion around CI. We don't want _more_ in-house build-servers to maintain when we're moving towards CI-aaS with most teams.